packages/docs/docs/getting-started/media.mdx

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_position: 13
+slug: /media
+---
+
+import AnimationPlayer from '@site/src/components/AnimationPlayer';
+
+# Media
+
+In this chapter we will take a look at all the different types of media Motion
+Canvas lets you add to your animations.
+
+## Audio
+
+One type of media Motion Canvas supports is audio. To add an audio track to your
+animation you would need to edit your `project.ts` file. First we need to import
+the audio file we want to use as follows:
+
+```ts
+// my-animation/
+// ├─ audio/
+// │  └─ voice.mp3
+// ├─ src/
+// ...
+import audio from '../audio/voice.mp3';
+```
+
+One imported we now need to let Motion Canvas know that we want to use this
```

Comment:
"Once imported, we need to let Motion Canvas know that we want to use this audio file." (One -> Once, remove "now")